src/libraries/System.Threading.AccessControl/src/System.Threading.AccessControl.csproj

```diff
@@ -21,7 +21,7 @@ System.Security.AccessControl.SemaphoreSecurity</PackageDescription>
     <IsPartialFacadeAssembly Condition="$([MSBuild]::GetTargetFrameworkIdentifier('$(TargetFramework)')) == '.NETFramework'">true</IsPartialFacadeAssembly>
     <GeneratePlatformNotSupportedAssemblyMessage Condition="'$(IsPartialFacadeAssembly)' != 'true' and '$(TargetsWindows)' != 'true'">SR.PlatformNotSupported_AccessControl</GeneratePlatformNotSupportedAssemblyMessage>
   </PropertyGroup>
-  <ItemGroup Condition="'$(TargetsWindows)' == 'true'">
+  <ItemGroup Condition="'$(IsPartialFacadeAssembly)' == 'true' or '$(TargetsWindows)' == 'true'">
```

Comment:
Why do we need this for the facade build?

Reply:
I don't know. I'm just preserving the existing behavior.

Reply:
@carlossanlop added this in https://github.com/dotnet/runtime/commit/66decde8e79ddd547569e9e305719f4130f3283c. Carlos do you know why this was added for the .NETFramework façade assembly as well?

Reply:
If you can remove it and it still compiles, go ahead.  My guess is it was just a quirk of the conditions previously present, we have a lot of this stuff across the repo where folks accidentally include some unused internal API or resources.